### DISCUSSION TITLE
Multiple Extruders

### DISCUSSION THREAD

**Discussion by Cyberblade2k** [OrcaSlicer/OrcaSlicer#828] [Ideas]
I have a Tenlog TL D3 Pro with dual extruders.  I would like to be able to select which extruder is doing model / color or supports / infill etc.  Currently, I have to go into the GCode and fix it myself.

Thanks

**Comment by hliebscher** [comment]
if you add a second color than switch between T0 and T1  but only for color ...

**Reply by Cyberblade2k** [reply]
Thanks
